httpclient5/src/test/java/org/apache/hc/client5/http/psl/TestPublicSuffixMatcher.java

```diff
@@ -40,8 +42,7 @@
 class TestPublicSuffixMatcher {
 
     private static final String SOURCE_FILE = "suffixlistmatcher.txt";
-    private static final String PUBLIC_SUFFIX_LIST_FILE = "mozilla/public-suffix-list.txt";
```

Comment:
@massdosage That notation should work on all platforms. I am quite sure of it. This is not the right fix.

Reply:
I've made that change here now.